bugsnag-android-performance/src/main/kotlin/com/bugsnag/android/performance/internal/processing/ImmutableConfig.kt

```diff
@@ -23,6 +24,7 @@ internal data class ImmutableConfig(
     val endpoint: String,
     val autoInstrumentAppStarts: Boolean,
     val autoInstrumentActivities: AutoInstrument,
+    val enabledMetrics: EnabledMetrics,
     val autoInstrumentRendering: Boolean,
```

Comment:
This field should be removed, since it's being replaced by `enabledMetrics.rendering` and `ImmutableConfig` is an internal class